src/opm/output/eclipse/Summary.cpp

```diff
@@ -1818,8 +1818,26 @@ namespace Evaluator {
 
 void reportUnsupportedKeywords(std::vector<Opm::SummaryNode> keywords)
 {
-    std::sort(keywords.begin(), keywords.end());
-    auto uend = std::unique(keywords.begin(), keywords.end());
+    // Sort by location first, then keyword.
+    auto loc_kw_ordering = [](const Opm::SummaryNode& n1, const Opm::SummaryNode& n2) {
+        if (n1.location() == n2.location()) {
```

Comment:
I would have written this with the `else` blocks:
```C++
if (n1.location() == n2.location())
     return ...

if ( )
     return 

return;
```

Reply:
Amended and force pushed (assuming you meant *without*, not *with*).

Reply:
Ahhh dang - you kept the braces; naught boy!

Reply:
Sorry, it did not really cross my mind! Not intending to be naughty.

Reply:
:smile: